internal/admin/main.tf

```diff
@@ -0,0 +1,75 @@
+# Copyright 2018 The Go Cloud Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+terraform {
+  backend "gcs" {
+    bucket = "go-cloud-tfstate"
+  }
+
+  required_version = "~>0.11"
+}
+
+provider "github" {
+  version      = "~> 1.3"
+  organization = "google"
+  token        = "${var.github_token}"
+}
+
+provider "google" {
+  version = "~> 1.15"
+}
+
+module "go_cloud_repo" {
+  source       = "./repository"
+  name         = "go-cloud"
+  description  = "A library and tools for open cloud development in Go."
+  homepage_url = "https://godoc.org/github.com/google/go-cloud"
```

Comment:
godoc.org/gocloud.dev

Reply:
Done. And this is why I like having code-reviewed configurations. 🙂